src/johnsession.h

```diff
@@ -35,8 +35,8 @@ class JohnSession
     int nbOpenMPThreads() const;
     void setNbOpenMPThreads(int nbOpenMPThreads);
 
-    int nbProcess() const;
-    void setNbProcess(int nbProcess);
+    int nbForkProcess() const;
+    void setNbForkProcess(int nbProcess);
```

Comment:
Get/set method names must always match the member name. Also change arg to something more appropriate.